src/controller.cpp

```diff
@@ -452,6 +443,7 @@ int controller::run(int argc, char * argv[]) {
 			refresh_on_start = true;
 			break;
 		case 'e':
+			silent = true; // export will be on StdOut
```

Comment:
The comment puzzled me for a minute because I couldn't understand how setting `silent` to `true` can redirect any output. Then I realized that what the comment meant is "disable logging to stdout *because export will be printed to stdout*". Can you use that phrasing, or something similarly verbose?

Reply:
Yes sure. I thought I keep it short, but I admit it's too short. :-)